tests/unit/xoopsLib/Xoops/Core/XoopsTplTest.php

```diff
@@ -38,16 +41,27 @@ public function testContracts()
         $this->assertInstanceOf('\Smarty', $this->object);
     }
 
+    public function normalize_path($path)
+    {
+        return str_replace('\\','/',$path);
+    }
+    
     public function test__construct()
     {
-        $xoops = \Xoops::getInstance();
         $this->assertSame('{', $this->object->left_delimiter);
         $this->assertSame('}', $this->object->right_delimiter);
-        $this->assertTrue(in_array(\XoopsBaseConfig::get('themes-path'). DIRECTORY_SEPARATOR, $this->object->getTemplateDir()));
-        $this->assertSame(\XoopsBaseConfig::get('var-path') . '/caches/smarty_cache'. DIRECTORY_SEPARATOR, $this->object->getCacheDir());
-        $this->assertSame(\XoopsBaseConfig::get('var-path') . '/caches/smarty_compile' . DIRECTORY_SEPARATOR, $this->object->getCompileDir());
+        $this->assertTrue(in_array(
+                            $this->normalize_path(\XoopsBaseConfig::get('themes-path')).'/',
+                            array_map(array($this,'normalize_path'), $this->object->getTemplateDir())));
+        $this->assertSame($this->normalize_path(\XoopsBaseConfig::get('var-path')) . '/caches/smarty_cache/',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\XoopsBaseConfig' in method 'test__construct'.](https://www.codacy.com/app/richard_3/XoopsCore/file/2444115150/issues/source?bid=3194562&fileBranchId=3245574#l56)